### PR TITLE
Replace leading slash in some relURL

### LINKED ISSUE #158: Leading slash in relative paths
# Observations
## Owl Carousel
The path in `resources.Get` in the following two codeblocks are different.
https://github.com/victoriadrake/hugo-theme-introduction/blob/e18c5c54295eaafdd8483e8ede487538866166d9/layouts/partials/js/owlCarousel.html#L1-L4
https://github.com/victoriadrake/hugo-theme-introduction/blob/e18c5c54295eaafdd8483e8ede487538866166d9/layouts/partials/css/owlCarousel.html#L1-L4
## Assets path and missing Font Awesome icons
In #85, assets weren't loading.  In https://github.com/victoriadrake/hugo-theme-introduction/pull/107#issuecomment-442518417, it's reported that Font Awesome icons are missing.  I've experienced that problem during [my test builds on GitLab](https://gitlab.com/VincentTam/introduction/-/jobs/).
https://github.com/victoriadrake/hugo-theme-introduction/blob/6c090cc278c91c718b8269ba9c3261282a3f23d9/scripts/build_docs.sh#L5
First, observe that the deploy target base URL contains a subpath `/hugo-theme-introduction`. The leading `/` might, in some cases, get interpreted as the root of the domain.  This idea is supported in [Hugo's Common Permalink Issues](https://github.com/gohugoio/hugoThemes#common-permalink-issues).

In https://github.com/victoriadrake/hugo-theme-introduction/commit/5895d84506e343b20d34d76c51edcc879602d666, which aimed at resolving the later issue, @hanzei removed the leading `/` in the Go code in SASS file's font path.

Sometimes the leading `/` might work, as https://github.com/gohugoio/hugo/issues/4733#issuecomment-390200427 suggests.  However, such approach isn't quite portable, espeically in case of subpath in base URL and/or multiple languages.

# Proposal

Implement the removal of leading `/` in 5895d84506e343b20d34d76c51edcc879602d666 for `js`, `css`, `sass` and `vendor`.

# See also

In [Hugo's documentation](https://gohugo.io/hugo-pipes/resource-from-template/#readout) containing [`resources.Get`](https://gohugo.io/hugo-pipes/introduction/#from-file-to-resource), every example contain a path that doesn't start with `/`.

### PR DESCRIPTION
**Pull request by VincentTam**
Resolves #158 with the command

```sh
for f in `git grep -Il 'resources.Get "/'`; do
  sed -i 's/"\/\(js\|css\|sass\|vendor\)/"\1/g' $f
done
```

You may check that all leading slashes have been removed after running the above command by

```sh
git grep -Ie '"/\?\(js\|css\|sass\|vendor\)' -- layouts
```

Working example: again, my demo Staticman GitLab Pages at https://vincenttam.gitlab.io/introduction ([source](https://gitlab.com/VincentTam/introduction/) on the `staticman` branch).

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -132,11 +132,11 @@ <h3 class="subtitle is-3 has-text-centered top-pad">
         {{ partial "js/owlCarousel.html" . }}
 
         {{ if .Site.Params.home.localTime }}
-        {{ $momentjs := resources.Get "/vendor/momentjs/moment.min.js" }}
-        {{ $momentTimezone := resources.Get "/vendor/momentjs/moment-timezone.min.js" }}
-        {{ $momentTimezoneWithData := resources.Get "/vendor/momentjs/moment-timezone-with-data-2012-2022.min.js" }}
-        {{ $initMomentjs := resources.Get "/js/initMoment.js" | resources.ExecuteAsTemplate "js/initMoment.tmp.js" . }}
-        {{ $bundleMoment := slice $momentjs $momentTimezone $momentTimezoneWithData $initMomentjs | resources.Concat "/js/bundleMoment.js" | fingerprint }}
+        {{ $momentjs := resources.Get "vendor/momentjs/moment.min.js" }}
```
Comment (hanzei) [inline layouts/index.html]: I double this is part of the problem. But I'm fine with changing it.

Comment (VincentTam) [reply layouts/index.html]: @hanzei Honestly speaking, I've no idea what Moment JS does.  I'm just replacing everything that starts with a forward slash according to the Hugo experts' recommendations.

Comment (hanzei) [reply layouts/index.html]: I was not referring to Moment JS. My point was that I don't think there is a difference between 
```
resources.Get "vendor/some/path"
```
and
```
resources.Get "/vendor/some/path"
```

Comment (VincentTam) [reply layouts/index.html]: @hanzei Thanks for clarification.  In practice, you're right.  The screenshots that I've taken about my PR for nested Staticman comments were made before this PR.

**Pull request merged**